packages/client/components/GitHubScopingSearchFilterMenu.tsx

```diff
@@ -26,6 +26,10 @@ const StyledCheckBox = styled(Checkbox)({
 })
 const StyledMenuItemLabel = styled(MenuItemLabel)({})
 
+const StyledMenu = styled(Menu)({
+  maxWidth: '100%'
```

Comment:
+1 i tried commenting this out & it looked the same. any steps to reproduce the problem?

Reply:
If a project fullPath is particularly long, it gets truncated without setting `maxWidth` to 100%. 
Without `maxWidth: '100%'`:
![before](https://user-images.githubusercontent.com/39854876/162191709-44c8c903-839a-4b75-8c15-3b4903eb1b11.png)

With `maxWidth: '100%'`:
![after](https://user-images.githubusercontent.com/39854876/162191738-5df63331-b260-4dff-9a80-909753d9d262.png)